app/views/lists/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<h1> Lists </h1>
+
+<table>
+  <thead>
+    <tr>
+      <th>Title</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% if @lists %>
+      <% @lists.each do |list| %>
+        <tr>
+          <td>
+            <%= link_to list.title, list_path(list.id) %>
+          </td>
+          <td>
+            <%= link_to 'Edit this list', edit_list_path(list.id) %>
+          </td>
+          <td>
+            <%= link_to 'Destroy this list', list_path(list.id), method: :delete %>
+          </td>
+        </tr>
+      <% end %>
+    <% else %>
+      <tr>
+        <td>
+          There are currently no lists for the user <%= current_user.email %>
```

Comment:
since this default value for email is an empty string, would it be good to define a fallback for users who have yet to define their email?

Reply:
Email was required when the user signs up, so I assumed everyone has an email address in the database. Devise's `authenticate_user!` method will have the user redirect_to log in page if they are not logged in or signed up for the application.